features/jobs.js

```diff
@@ -3,6 +3,8 @@ const cooldown = require("./cooldown").default;
 const jobs = {
   tags: ["forhire", "for hire", "hiring", "remote", "local"],
   handleMessage: ({ msg, user }) => {
+    if (msg.author.bot) return;

```

Comment:
Is this strictly necessary?

Reply:
Yes? What benefit would we get if we had bots messaging/triggering each other?

Reply:
This is necessary because the bot may post a message in the `#jobs` channel that does not contain these tags so it would receive a warning from itself